pybabylonjs/show.py

```diff
@@ -32,17 +30,55 @@ def __init__(self):
     @classmethod
     def point_cloud(
         self,
-        data: dict,
-        time: Optional[bool] = False,
-        classes: Optional[bool] = False,
-        topo: Optional[bool] = False,
+        uri: Optional[str] = "uri",
+        data: Optional[dict] = {},
+        source: Optional[str] = "cloud",
+        mode: Optional[str] = "default",
         **kwargs,
     ):
-        d = {"classes": classes, "time": time, "topo": topo, "data": data}
-        if topo:
-            img = create_mapbox_image(data, **kwargs)
+        """
+        Returns a point cloud visualization widget
+
+        :param uri: when source is "cloud" or "local" specify the URI for the TileDB array
+        :param data: when source="dict" this dictionary contains the points to be visualized: {"X", "Y", "Z", "Red", "Green", "Blue"}
+        :param source: location of the data to be visualized, one of "cloud", "local" or "dict"
+        :param mode: sub-type of the visualization, one of "default", "time", "classes" or "topo"
+
+        :Keyword Arguments:
+
+        * **bbox** - ...
+        * **classes** - ...
+        * **time_offset** - ...
+        *
+
+
+        """
+
+        if source == "dict":
+            data = check_point_cloud_data_dict(mode, data)
+        if source == "local":
+            data = check_point_cloud_data_local(mode, uri, kwargs)
+        if source == "cloud":
+            # not implemented yet
```

Comment:
Do you have the code for this? If so, then lets add it to this PR so we are all using the same API